definition_tooling/api_errors.py

```diff
@@ -3,29 +3,37 @@
 
 These errors can not be overridden by the data product definition itself.
 """
-from typing import Optional
-
 from pydantic import BaseModel, Field
 
 
 class BaseApiError(BaseModel):
     __status__: int
 
+    @classmethod
+    def get_response_spec(cls):
+        return {"model": cls}
+
 
-class ApiError(BaseApiError):
+class ApiErrorOne(BaseApiError):
     type: str = Field(..., title="Error type", description="Error identifier")
     message: str = Field(..., title="Error message", description="Error description")
 
 
-class Unauthorized(ApiError):
+class ApiErrorTwo(BaseApiError):
+    @classmethod
+    def get_response_spec(cls):
+        return {"model": cls, "content": {"text/plain": {}, "text/html": {}}}
```

Comment:
These names are not particularly descriptive, could you come up with better ones?